server/liveblog/advertisements/advertisements.py

```diff
@@ -25,6 +25,10 @@ class AdvertisementsResource(Resource):
         },
         'meta': {
             'type': 'dict',
+            'mapping': {
+                'type': 'dict',
```

Comment:
in the mapping this type should be object we send it to elastic as it is, and if you want this not to be indexed at all, you should use `'enabled': false` rather than `not_analyzed` which avoid analyzer but indexes the whole thing.
[more info](https://www.elastic.co/guide/en/elasticsearch/reference/current/object.html)